eng/pipelines/libraries/execute-linker-tests-steps.yml

```diff
@@ -0,0 +1,4 @@
+  steps:
+    # Execute tests
+    - script: $(Build.SourcesDirectory)$(dir)build$(scriptExt) -ci -arch x64 $(_osParameter) -s libs.tests -c release /p:LinkerTests=true $(_officialBuildParameter) $(_crossBuildPropertyArg) /bl:$(Build.SourcesDirectory)/artifacts/log/LinkerTests.binlog
```

Comment:
Why is configuration always release? Should it be using `$(_BuildConfig)` instead?

Reply:
I can change this if needed, but it is always release because for linker we ideally always want to run and publish with release bits so all of our verticals use Release configuration.

Reply:
Yeah I imagined that, but I rather be consistent with making the BuildConfig passed down to the template be the source of truth, since that is how it is in all the platform-matrix template expansion.